assets/css/event.css

```diff
@@ -1,37 +1,33 @@
-@import url('https://fonts.googleapis.com/css2?family=STIX+Two+Math&display=swap');
-*{
-    margin:0;
-    padding:0;
-    box-sizing:border-box;
-    font-family:"Poppins", sans-serif;
-    
-  }
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
 /* variables */
-    html{
-      --dull-white:#ced4e2;
-      --dull-black:#141516;
-      --purple:#8456c4;
-      --white-color: #ffffff;
-      --greyish:#2b2a2A;
-      --green-color:#62aa54;
+html {
```

Comment:
Refer [this](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/assets/css/experience.css) for defining the variables for the CSS file.

Instead of using the names as --green-color make it something more interpretable. For example: --card-bg-color